### DISCUSSION TITLE
Dashboard not displaying graphs and metrics.

### DISCUSSION THREAD

**Discussion by Andreas-White** [DependencyTrack/dependency-track#3144] [Q&A]
Hello, we are running currently dependency track on all our services for managing and tracking vulnerabilities and licenses. Unfortunately, since the end of September the dashboard doesn't display any graphs or metrics. We are currently on v4.8.2. Moreover, we're getting a browser error indicating that the api-server is not returning vulnerabilities. 
`TypeError: Cannot read properties of undefined (reading 'vulnerabilities')`. Some more details: All the projects are loaded normally (around 31k in total), as well as vulnerabilities section and licenses. Any ideas what could be causing that or anyone had similar experience?

**Comment by YaseenBabaShaik** [comment]
Hi How can we assign a static ip and port number to Dependency track from a pool of subnet available instead of using local host:8080
When deploying dependency track in aws and browse with the public ip in local browser DP track page loads but it blocks the http with not secure icon.Please help any suggestions I tried all possible ways like changing configuration of browser ,ports..

**Reply by Poremich** [reply]
any updates?

**Comment by Jason-Locke-Ionos** [comment]
I see the same in one of our instances (not all of them). It happened with the latest 4.12 and also we see it in 4.13. It could be data-related in some way but that's only a guess.
